web3.js/src/epoch-schedule.ts

```diff
@@ -0,0 +1,65 @@
+const MINIMUM_SLOT_PER_EPOCH = 32;
+
+// Returns the number of trailing zeros in the binary representation of self.
+function trailingZeros(n: number) {
+  let trailingZeros = 0;
+  while ((n & 1) == 0) {
```

Comment:
This will not stop looping if n is 0
```suggestion
  while (n > 1) {
```